painters/data_provider.py

```diff
@@ -269,18 +277,23 @@ def pairs_generator(X, y, batch_size, pair_func, num_groups):
 
 
 def _split_into_groups(y, num_groups):
+    # num_groups = 32
```

Comment:
This comment is a little confusing -- `num_groups` is going to be the value of the argument, perhaps place this comment a level up?